src/lib/github/apiActions/apiActions.ts

```diff
@@ -0,0 +1,62 @@
+"use server";
+
+import {
+  BaseGithubApiActions,
+  GitHubPaginatedResponse,
+} from "./baseApiActions";
+
+const githubApiClient = BaseGithubApiActions();
+
+export async function getAuthenticatedGitHubUser(): Promise<GitHubUser> {
+  return githubApiClient.get<GitHubUser>("/user");
+}
+
+export async function getAllReposForUsername(
+  owner: string,
```

Comment:
Whenever there's more than one variable, it's a good idea to pass an object instead of multiple inputs. That way you can pass the inputs in any order / it is no longer possible to pass them in the wrong order.